mw-webapp/src/component/text/Text.tsx

```diff
@@ -0,0 +1,45 @@
+import {ReactElement} from "react";
+
+/**
+ * Params for {@link Text}
+ */
+interface TextProps {
+
+  /**
+   * Displayed values
+   */
+  value: string | number;
+
+  /**
+   * Displayed placeholder
+   */
+  placeholder: string;
+
+  /**
+   * Function to render value
+   */
+  renderValue: (value: string | number) => ReactElement;
+
+  /**
+   * Function to render placeholder
+   */
+  renderPlaceholder: (placeholder: string) => ReactElement;
+}
```

Comment:
Add prop type (for the future) - example you can find in the component button/ It will be needed for using different styled text in different places. You can create only one type right now - because design is not ready

Reply:
Added props text to the component. To display different types of text, as well as the size of the text. Depending on what type of test is needed